media/stylus/includes/vars.styl

```diff
@@ -173,7 +173,7 @@ $selector-heading-font-fallback = 'html[data-ffo-opensanslight="false"]:not(.no-
 paths
 ====================================================================== */
 $path-to-assets = '/media/';
-$path-to-images = $path-to-assets + 'img/';
+$path-to-images = '../../static/img/';
 $logo-sprite-url = $path-to-images + 'logo_sprite.svg?2014-01';
```

Comment:
Definitely. That's the plan down the line. At the moment there are some uses ([example](https://github.com/mozilla/kuma/blob/master/media/stylus/base/fonts/opensans.styl#L5-L6)) where `$path-to-assets` needs to be more explicit.

A great point though. I'll be sure to remember this once everything is moved to static.